Makefile

```diff
@@ -20,11 +20,9 @@ VERSION=$(shell cat ./VERSION)
 
 GOFLAGS=-ldflags "-X github.com/edgexfoundry/edgex-go.Version=$(VERSION)"
 
-build:
+build: $(MICROSERVICES)
 	go build ./...
```

Comment:
Isn't line 24 redundant now with the $(MICROSERVICES) block above it?

Reply:
No. There is a subtle difference. 
For example `support/scheduler-client/` is not used by any microservice, if we don't add the `go build ./...` we will not test if it compiles at least.